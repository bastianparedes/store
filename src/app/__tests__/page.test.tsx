import React from 'react';

import { render } from '@testing-library/react';

import Page from '../page';

it('should render', () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
