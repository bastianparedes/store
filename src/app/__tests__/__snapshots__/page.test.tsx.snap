// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Bienvenido a Geek Store
    </h1>
    <p>
      Bienvenido a 
      <b>
        Geek Store
      </b>
      , tu destino definitivo para la compra y venta de de productos geek relacionados con videojuegos, juegos de mesa y anime. Sumérgete en nuestro extenso catálogo, que abarca desde los retro hasta los últimos lanzamientos de videojuegos, emocionantes juegos de mesa y una amplia colección de artículos de anime y manga. En Geek Store, nos enorgullece ofrecer productos auténticos y de alta calidad, respaldados por vendedores confiables, para que puedas expresar tu amor por la cultura geek con estilo. Nuestra comunidad vibrante te brinda la oportunidad de conectarte con otros aficionados, compartir experiencias y descubrir nuevas pasiones. Con una interfaz intuitiva y opciones de compra seguras, en Geek Store no solo adquieres productos, sino que también te sumerges en una experiencia única que celebra tu geek interior. ¡Descubre lo mejor del mundo geek con nosotros!
    </p>
    <button>
      Iniciar sesión
    </button>
  </div>
</div>
`;
